src/client/src/app/site/[site]/site-page.tsx

```diff
@@ -61,3 +58,23 @@ export const SitePage = ({ site, permissions }: SitePageProps) => {
     </>
   );
 };
+
+const mapSummaryData = (site: Site) => {
+  if (!site) {
+    return undefined;
+  }
+
+  const items: SummaryListItem[] = [];
+
+  items.push({
+    title: 'Address',
+    value: site.address.match(/[^,]+,|[^,]+$/g) || [], // Match each word followed by a comma, or the last word without a comma
```

Comment:
Does split not work here? Seems quite complex implementation

Reply:
split works but it removed comma at the end of each item which goes agains UI design. Got some other ideas how to split by comma by keep it in items?